var/spack/repos/builtin/packages/lammps/package.py

```diff
@@ -177,15 +177,14 @@ def build_voronoi(self):
                         'compute_voronoi_atom.h')
 
     def build(self, spec, prefix):
-        for pkg in self.supported_packages:
```

Comment:
That's true. Someone will have to test this out. I'm not sure if `with working_dir` stacks or not, but you're probably right.